### PR TITLE
[Sample] Update pre-load TFX::OSS sample

### PR DESCRIPTION
**Pull request by numerology**
Add a patch to enable visualization

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2476)

### CODE REVIEW

**Comment by numerology** [comment]
/hold cancel
Looks good. TB is working fine.

File: backend/src/apiserver/config/sample_config.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "file": "/samples/core/exit_handler/exit_handler.py.tar.gz"
   },
   {
-    "name": "[Sample] ML - TFX - Taxi Tip Prediction Model Trainer",
+    "name": "[Sample] Unified DSL - Taxi Tip Prediction Model Trainer",
```
Comment (IronPan) [inline backend/src/apiserver/config/sample_config.json]: do you mind move this next to xgboost sample ? the samples are loaded in reverse order and would be nice basic samples are at top.

**Comment by IronPan** [comment]
thanks for the quick turnaround.

File: backend/src/apiserver/config/sample_config.json (hunk 1 above)
Comment (numerology) [reply backend/src/apiserver/config/sample_config.json]: Sure. Thanks for the fixing!

**Pull request merged**